homeassistant/components/snapcast/media_player.py

```diff
@@ -84,6 +86,14 @@
             elif service == SERVICE_UNJOIN:
                 if isinstance(device, SnapcastClientDevice):
                     await device.async_unjoin()
+            elif service == SERVICE_SET_LATENCY:
+                if isinstance(device, SnapcastClientDevice):
+                    master = [
+                        e
+                        for e in hass.data[DATA_KEY]
+                        if e.entity_id == data[ATTR_ENTITY_ID]
+                    ]
+                    await device.async_set_latency(data.get(ATTR_LATENCY))
```

Comment:
Please use our new way of registering entity service handlers:
https://developers.home-assistant.io/docs/dev_101_services#entity-services

Reply:
Should I change all existing service calls also?

Reply:
It's ok to add a new handler just for this service and migrate the other services later.

If you want to clean them up now and can test that, it's ok too.

Reply:
Okay this is my first contribution so give me a bit to figure it out. Want the commits squashed or want a second commit for code fixes/cleanup?

Reply:
Please just add new commits after review has started. That makes it easier to track the changes.

We will squash when merging.